cmd/checks/components/components_check.go

```diff
@@ -69,6 +70,7 @@ func Add(root *cobra.Command) {
 	componentsCmd.Flags().StringVarP(&healthURLPrefix, "health-url", "u", "/system/health/v1", "Set dcos-diagnostics health url")
 	componentsCmd.Flags().StringVarP(&scheme, "scheme", "s", "http", "Set dcos-diagnostics health url scheme")
 	componentsCmd.Flags().IntVarP(&port, "port", "p", 1050, "Set TCP port")
+	componentsCmd.Flags().StringArrayVar(&excludeComponents, "exclude-components", nil, "Exclude components from health check")
```

Comment:
i think you need to use `StringArrayVarP`

Reply:
https://godoc.org/github.com/spf13/pflag#StringArrayVar
The difference is a shorthand which I wasnt planning on using.